### DISCUSSION TITLE
How to change the recorded files to MONO

### DISCUSSION THREAD

**Discussion by dalogue1** [Nachtzuster/BirdNET-Pi#134] [General]
Since the BirdNET-Pi only identifies off of a single microphone, and it makes the captured files larger,  how could the birdnet_recording.sh be modified to capture MONO instead of STEREO?

I believe the birdnet_recording.sh is recording using ffmpeg.  Would I just need to modify the FFMPEG_PARAMS string to change from STEREO to MONO?

Possibly change the following line to reduce the channels from 2 to 1.

>       FFMPEG_PARAMS+="-vn -thread_queue_size 512 -i ${i} -map ${MAP_ID}:a:0 -t ${RECORDING_LENGTH} -acodec pcm_s16le -ac 2 -ar 48000 file:${RECS_DIR}/StreamData/$(date "+%F")-birdnet-RTSP_${RTSP_STREAMS_STARTED_COUNT}-$(date "+%H:%M:%S").wav "

changes to 

>       FFMPEG_PARAMS+="-vn -thread_queue_size 512 -i ${i} -map ${MAP_ID}:a:0 -t ${RECORDING_LENGTH} -acodec pcm_s16le -ac 1 -ar 48000 file:${RECS_DIR}/StreamData/$(date "+%F")-birdnet-RTSP_${RTSP_STREAMS_STARTED_COUNT}-$(date "+%H:%M:%S").wav "

-ac 2 (stereo) is changed to -ac 1 (mono)

**Comment by alexbelgium** [comment]
Hi, I think that's what this options does `Audio Channels:  Set Channels to the number of channels supported by your sound card. 32 max.` in the advanced settings

**Reply by dalogue1** [reply]
Thank you.  I changed the option to 1 instead of 2 and the file is half the size.
